flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/jdbc/MysqlPropertiesConverter.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.flink.connector.jdbc;
+
+public class MysqlPropertiesConverter extends JdbcPropertiesConverter {
+
+  public static final MysqlPropertiesConverter INSTANCE = new MysqlPropertiesConverter();
+
+  private MysqlPropertiesConverter() {}
+
+  @Override
+  public String driverName() {
+    return "com.mysql.jdbc.Driver";
```

Comment:
There may multi Mysql drivers with different names, seems we need a more general solution to handle jdbc-driver

Reply:
To ensure compatibility with MySQL 5.x, the Flink JDBC catalog uses com.mysql.jdbc.Driver as the JDBC driver.